styles/settings-view.less

```diff
@@ -7,6 +7,7 @@
 
 .settings-view {
   display: flex;
+  overflow: auto;
```

Comment:
Yeah right, if very narrow (smaller than `180px`). I was thinking that if your pane is that narrow, you don't really want to open the settings in it. But be able to scroll everything might be nice too because it gives more space for the panels.

Let's merge this. 👍 @frantic1048 Thanks.